xbmc/ContextMenuItem.cpp

```diff
@@ -53,7 +53,7 @@ bool CContextMenuItem::Execute(const std::shared_ptr<CFileItem>& item) const
     return false;
 
   ADDON::AddonPtr addon;
-  if (!CServiceBroker::GetAddonMgr().GetAddon(m_addonId, addon, ADDON::ADDON_UNKNOWN,
+  if (!CServiceBroker::GetAddonMgr().GetAddon(m_addonId, addon, ADDON::AddonType::ADDON_UNKNOWN,
```

Comment:
Because we moved to an enum class, the leading `ADDON_` is kinda unnecessary as the class name is required, and if removed I think it would reduce redundancy:

```
ADDON::AddonType::UNKNOWN
```

Reply:
We have learned that using simple words like YES, NO, OK, CANCEL for enum class members can lead to compile errors, so ended with prefixing all members just to be on the safe side. UNKNOWN feels a bit like a candidate for , but I admit that I did not actually try.

Reply:
Valid, one of the labels is ADDON_MAX